sw/device/tests/aon_timer_wdog_bite_reset_test.c

```diff
@@ -33,8 +33,8 @@ static void config_wdog(const dif_aon_timer_t *aon_timer,
   uint32_t bite_cycles =
       aon_timer_testutils_get_aon_cycles_from_us(bite_time_us);
 
-  LOG_INFO("Wdog will bark after %u us and bite after %u us", bark_time_us,
-           bite_time_us);
+  LOG_INFO("Wdog will bark after %u us and bite after %u us",
+           (uint32_t)bark_time_us, (uint32_t)bite_time_us);
```

Comment:
Do we allow `%u` with `uint32_t`? That's not quite right with `printf()`, for example (but might work for OpenTitan if `uint32_t` happens to be defined as `unsigned int`).

Reply:
I think we do, here is an [example](https://github.com/lowRISC/opentitan/blob/f999d5515d774136880a666677f370f694c0e801/sw/device/lib/testing/test_framework/freertos_port.c#L70)

Reply:
Yes, we allow `%u`. Our printf assumes that uinsigned int is u32 (which is a safe assumption on any platform we would ever care to port this code to regardless).

Reply:
Cool, thanks for clearing that up.